spec/unit/installer/xcode/pods_project_generator/target_installer_spec.rb

```diff
@@ -54,7 +54,7 @@ class PodsProjectGenerator
             @pod_target.stubs(:uses_swift?).returns(true)
             @installer.send(:add_target)
             @installer.send(:native_target).resolved_build_setting('SWIFT_OPTIMIZATION_LEVEL').should == {
-              'Release' => nil,
+              'Release' => '-Owholemodule',
```

Comment:
As for the "older Xcodes" concern: if people want to use an outdated Xcode version, shouldn't they also use the equally outdated CocoaPods version that supports that Xcode version?